### PR TITLE
Sequence.last changed to PositiveBigIntegerField

### PR DESCRIPTION
**Pull request by petrprikryl**
Tests passed on all databases.

### CODE REVIEW

**Comment by petrprikryl** [comment]
https://github.com/aaugustin/django-sequences/issues/32

File: README.rst [hunk 1]
```diff
@@ -434,6 +434,11 @@ Build and publish the new version:
 
 Changelog
 =========
+2.7
+---
```
Comment (adamchainz) [inline README.rst:438]: I think it's normal to only add version numbers at release time - at least that's my preference.

File: README.rst [hunk 2]
```diff
@@ -434,6 +434,11 @@ Build and publish the new version:
...
+
+* ``Sequence.last`` field changed from ``PositiveIntegerField`` to ``PositiveBigIntegerField``
```
Comment (adamchainz) [inline README.rst:440]: I think we should have a few more words covering that this requires a migration, and thatit may take a while on projects with many sequences. I'm not sure there's any general purpose advice we can give to users but we should highlight the potential for locking during deployment.

File: README.rst (hunk 1 above)
Comment (petrprikryl) [reply README.rst]: :+1: Removed.

File: README.rst (hunk 2 above)
Comment (petrprikryl) [reply README.rst:438]: Added detailed info. Did my best :slightly_smiling_face:

**Comment by aaugustin** [comment]
Given that django-sequences is very stable, I made a release.

**Comment by adamchainz** [comment]
Nice work all!

And good tweak of the changelog in 83f4a0f2d99954e43d9757f1cd72081fb39a5b2f @aaugustin - I was going to comment that it's best to avoid "aggressive" as it anthropomorphizes technology.

**Pull request merged**